src/components/tables/cells/EntityName.tsx

```diff
@@ -31,18 +38,27 @@ function EntityName({ name, showEntityStatus }: Props) {
             >
                 {showEntityStatus ? <EntityStatus name={name} /> : null}
 
-                <Typography
-                    sx={
-                        belowMd
-                            ? {
-                                  overflowWrap: 'break-word',
-                                  wordBreak: 'break-all',
-                              }
-                            : undefined
-                    }
+                <Tooltip
```

Comment:
Added a tooltip in case a user does not know what will happen when the link is clicked